test/e2e-auto-tls-tests.sh

```diff
@@ -174,7 +174,8 @@ function delete_dns_record() {
 # Skip installing istio as an add-on
 # Temporarily increasing the cluster size for serving tests to rule out
 # resource/eviction as causes of flakiness.
-initialize "$@" --skip-istio-addon --min-nodes=4 --max-nodes=4
+# Pin to 1.18 since scale test is super flakey on 1.19
+initialize "$@" --skip-istio-addon --min-nodes=4 --max-nodes=4 --cluster-version=1.18.16-gke.502
```

Comment:
You're welcome to have this very specific version, but `1.18.16` and `1.18` should also work, if you desire something slightly less brittle.

Reply:
oh good to know thanks